### PR TITLE
Handle duplicates by only reading in first active site in PDB file

### PR DESCRIPTION
**Pull request by greyson9**
modified io.py to only read the first chain of each PDB so that there are no duplicate sites in sequence

### CODE REVIEW

File: hw2skeleton/cluster.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from .utils import Atom, Residue, ActiveSite
+from hw2skeleton.utils import Atom, Residue, ActiveSite
```
Comment (tlnagy) [inline hw2skeleton/cluster.py:1]: This is unnecessary.

File: hw2skeleton/io.py [hunk 2]
```diff
@@ -18,11 +18,12 @@ def read_active_sites(dir):
 
         active_sites.append(read_active_site(filepath))
 
-    print("Read in %d active sites"%len(active_sites))
+    print("Read in %d active sites" % len(active_sites))
 
     return active_sites
 
 
+# modified to only read the first chain of a PDB
```
Comment (tlnagy) [inline hw2skeleton/io.py:26]: This information should be in your commit message, not in a comment.

File: hw2skeleton/io.py [hunk 3]
```diff
@@ -58,14 +64,17 @@ def read_active_site(filepath):
 
                 # make a new residue if needed
                 if residue_number != r_num:
-                    residue = Residue(residue_type, residue_number)
+                    residue = Residue(residue_type, str(residue_number))
```
Comment (tlnagy) [inline hw2skeleton/io.py:67]: This isn't needed any more because of my changes in 376deba666f11b429567e1f236fa72d29df186c0

File: hw2skeleton/pdb_to_fasta.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import sys
```
Comment (tlnagy) [inline hw2skeleton/pdb_to_fasta.py:1]: Not everyone wants to go the FASTA route so I don't think we should add this file to this PR.

File: hw2skeleton/utils.py [hunk 5]
```diff
@@ -1,32 +1,35 @@
...
 
+
 class Atom:
     """
     A simple class for an amino acid residue
     """
 
-    def __init__(self, type):
-        self.type = type
+    def __init__(self, type_var):
```
Comment (tlnagy) [inline hw2skeleton/utils.py:9]: Why make this change? Is there a conflict with the name?

File: test/test_io.py [hunk 6]
```diff
@@ -3,8 +3,8 @@
 import os
 
 @pytest.mark.parametrize("filename,names,numbers", [
-    ("276.pdb", ["HIS", "HIS", "HIS", "HIS", "ASP"], [55, 57, 201, 230, 301]),
-    ("4629.pdb", ["ASP", "THR", "ARG", "SER", "LYS", "TYR", "SER", "ASN", "ASP"], [10, 14, 41, 118, 151, 157, 176, 177, 180]),
+    ("276.pdb", ["HIS", "HIS", "HIS", "HIS", "ASP"], ['55', '57', '201', '230', '301']),
+    ("4629.pdb", ["ASP", "THR", "ARG", "SER", "LYS", "TYR", "SER", "ASN", "ASP"], ['10', '14', '41', '118', '151', '157', '176', '177', '180']),
 ])
```
Comment (tlnagy) [inline test/test_io.py:8]: Residue number should be a integer, not a string. See my change in 376deba666f11b429567e1f236fa72d29df186c0.

**Review by tlnagy — CHANGES_REQUESTED** [review]
Hey Greyson, thanks for submitting a PR! We'll need to make some changes before I merge into in. I've commented on your code below. You can push more commits to this branch to fix things. I'll squash everything before we merge.

cheers,
-tamas

File: test/test_io.py (hunk 6 above)
Comment (tlnagy) [reply test/test_io.py:8]: Also, can you add a test for your code? For example, load a PDB with known duplicate active sites and make sure that it is read in correctly.
